model/task.js

```diff
@@ -198,10 +196,10 @@ module.exports = function(app, callback) {
 						chromeLaunchConfig: app.config.chromeLaunchConfig || {},
 						headers: task.headers || {},
 						log: {
-							debug: pa11yLog,
-							error: pa11yLog,
-							info: pa11yLog,
-							log: pa11yLog
+							debug: model.pa11yLog(),
+							error: model.pa11yLog(),
+							info: model.pa11yLog(),
+							log: model.pa11yLog()
```

Comment:
As you are creating a closure over `this.id` with `pa11yLog`, why not pass `task.id` into `modeo.pa11yLog` and it returns a logging function that uses that ID?

Reply:
Because I'm dumb, mostly.

That's done now (I think).